src/Model/AbstractParking.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use App\Model\ParkingInterface;
+use App\Model\VehicleInterface;
+use App\ValueObject\Money;
+
+abstract class AbstractParking extends Money implements ParkingInterface
+{
+    protected $numberOfPlaces;
+    protected $pricePerHour;
```

Comment:
Jeżeli używasz PHP 7.2 to dodawaj PHP Doc'a np. "/** @var string */", aby określić typ property.

Jeżeli używasz wyższej wersji, to określaj typy jawnie